core/sr-api-macros/tests/runtime_calls.rs

```diff
@@ -70,11 +70,45 @@ fn calling_native_runtime_signature_changed_function() {
 
 #[test]
 fn calling_wasm_runtime_signature_changed_old_function() {
-	let client = test_client::new_with_api_execution_strat(ExecutionStrategy::AlwaysWasm);
+	let client = test_client::new_with_execution_strategy(ExecutionStrategy::AlwaysWasm);
 	let runtime_api = client.runtime_api();
 	let block_id = BlockId::Number(client.info().unwrap().chain.best_number);
 
 	#[allow(deprecated)]
 	let res = runtime_api.function_signature_changed_before_version_2(&block_id).unwrap();
 	assert_eq!(&res, &[1, 2]);
 }
+
+#[test]
+#[should_panic]
```

Comment:
Please provide an expected message, see above for an example. (So we make sure that the panic is the one we expect).